superset/reports/commands/execute.py

```diff
@@ -77,11 +77,24 @@
 logger = logging.getLogger(__name__)
 
 
-def _get_user() -> User:
-    user = security_manager.find_user(username=app.config["THUMBNAIL_SELENIUM_USER"])
-    if not user:
-        raise ReportScheduleSelleniumUserNotFoundError()
-    return user
+def _get_user(report_schedule: ReportSchedule) -> User:
+    user_types = app.config["ALERT_REPORTS_EXECUTE_AS"]
+    for user_type in user_types:
+        if user_type == "selenium":
```

Comment:
what do you think about defining an `Enum` for this?

Reply:
I usually opt for `Literal` in the case of config flags, as using an `Enum` requires adding an import which is an added inconvenience. But let me see if mypy is ok assigning a string value to something that is extending `(str, Enum)`

Reply:
No luck 🙁 `Expected type 'List[ReportScheduleExecutor]', got 'List[str]' instead`. I made the change anyway, it does look better (and anyone who wants to use the literal value can do that if they want to).